sauce.browsers.js

```diff
@@ -0,0 +1,141 @@
+const sauceBrowsers = {
```

Comment:
Ok, cool. Just wondering if it was anything else. Did you consider moving it into a folder? The only thing I can think of is a possible future where we start moving more stuff out, but maybe that's premature optimisation.